Resources/Prototypes/Recipes/Construction/Graphs/structures/blast_door.yml

```diff
@@ -0,0 +1,110 @@
+﻿- type: constructionGraph
+  id: BlastDoor
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: frame1
+          completed:
+            - !type:SnapToGrid { }
+          steps:
+            - material: Plasteel
+              amount: 10
+              doAfter: 3
+
+
+    - node: frame1
+      entity: BlastDoorFrame
+      edges:
+        - to: frame2
+          conditions:
+            - !type:EntityAnchored
+              anchored: true
+          steps:
+            - material: Cable
+              amount: 5
+              doAfter: 2
+        - to: start
+          completed:
+            - !type:SpawnPrototype
+              prototype: SheetPlasteel1
+              amount: 10
+            - !type:DeleteEntity {}
+          conditions:
+            - !type:EntityAnchored
+              anchored: true
+          steps:
+            - tool: Welding
+              doAfter: 4
+
+    - node: frame2
+      edges:
+        - to: frame3
+          conditions:
+            - !type:EntityAnchored
+              anchored: true
+          steps:
+            - tag: DoorElectronics
+              name: Door Electronics
+              icon:
+                sprite: "Objects/Misc/module.rsi"
+                state: "door_electronics"
+        - to: frame1
+          completed:
+            - !type:SpawnPrototype
+              prototype: CableApcStack1
+              amount: 5
+          conditions:
+            - !type:EntityAnchored
+              anchored: true
+          steps:
+            - tool: Cutting
+              doAfter: 4
+            - tool: Prying
+              doAfter: 2
+
+    - node: frame3
```

Comment:
need an edge to frame2 so you can door elec -> oops i dont want to anymore without screwing and removing

Reply:
I'm not sure I got it right, but I did something )

Reply:
frame4 should go to frame3 not frame2, then remove the pry action
frame3 should have a prying action to go back to frame2

Reply:
If you go from frame4 to frame3, it breaks the whole deconstruction. everything works fine with frame2.

Reply:
disassembly is intentionally more complicated than assembly

Reply:
ok i guess